### PR TITLE
Feat/ss validation response improvement

### PR DESCRIPTION
**Pull request by nikhil-024**
# Odin version(x.y.z)

### CODE REVIEW

File: proto/dream11/od/dto/v1/service.proto [hunk 1]
```diff
@@ -22,8 +22,8 @@ message ServiceDefinition {
 }
 
 message ServiceIdentifier {
-  string name = 1;
-  string version = 2;
+  string service_name = 1;
```
Comment (dhavalmehta07) [inline proto/dream11/od/dto/v1/service.proto:25]: why this change? older names look good to me

File: internal/service/service.go [hunk 2]
```diff
@@ -50,14 +51,15 @@ func (e *Service) DeployService(ctx *context.Context, request *serviceProto.Depl
...
+				message += fmt.Sprintf("\n Component %s %s %s %s", compMessage.ComponentName, compMessage.ComponentAction, compMessage.ComponentStatus,compMessage.Error)
 			}
 			spinnerInstance.Prefix = fmt.Sprintf(" %s  ", message)
+			prevMessage = message
 			spinnerInstance.Start()
 		}
 	}
 
-	log.Info(message)
+	log.Info(prevMessage)
```
Comment (dhavalmehta07) [inline internal/service/service.go:62]: do we need this?

Comment (nikhil-024) [reply internal/service/service.go:62]: yes, this is required to print the last message because it is cleared before printing

File: proto/dream11/od/dto/v1/service.proto (hunk 1 above)
Comment (nikhil-024) [reply proto/dream11/od/dto/v1/service.proto:25]: we have used this new name in deployer proto files. is easier to change name here than in deployer as it might be used in multiple places.

**Review by dhavalmehta07 — APPROVED** [review]
